clouddriver-cloudfoundry/src/test/java/com/netflix/spinnaker/clouddriver/cloudfoundry/deploy/ops/AbstractCloudFoundryAtomicOperationTest.java

```diff
@@ -0,0 +1,61 @@
+/*
```

Comment:
FWIW it's totally fine to write tests in groovy if you find it easier -- not necessary either way though.

Reply:
At this point I really don't like Groovy much. Java is fine.